hotsos/cli.py

```diff
@@ -19,6 +20,18 @@
 )
 
 
+def is_snap():
+    return os.environ.get('SNAP_NAME', '') == 'hotsos'
+
+
+def get_os_id():
+    return distro.id() == 'ubuntu'
```

Comment:
This should just return distro.id() as the caller is checking that value, not true/false.

Reply:
Oh right, good catch.